VMBackup/main/fsfreezer.py

```diff
@@ -248,10 +238,23 @@ def freeze_safe(self,timeout):
             self.logger.log(error_msg, True, 'Error')
         return freeze_result,timedout
 
+    def releaseFileLock(self):
+        if (self.isAquireLockSucceeded == True):
+            try:
+                fcntl.lockf(self.file, fcntl.LOCK_UN)
+                self.file.close()
+            except:
+                pass
+        try:
+            os.remove("/etc/azure/MicrosoftRecoverySvcsSafeFreezeLock/SafeFreezeLockFile")
```

Comment:
Is delete really required ?

Reply:
yes if in case lock is not released then delete will rule out any possibility of starving of another process